modules/miner/miner_test.go

```diff
@@ -12,9 +15,13 @@ import (
 // TestMiner creates a miner, mines a few blocks, and checks that the wallet
 // balance is updating as the blocks get mined.
 func TestMiner(t *testing.T) {
+	directory := "TestMiner"
+
 	// Create the miner and all of it's dependencies.
 	s := consensus.CreateGenesisState()
-	g, err := gateway.New(":8900", s, "")
+
+	gDir := filepath.Join(tester.TempDir(directory), modules.GatewayDir)
```

Comment:
this pattern seems common enough that you might want to add a function in `tester` that does the `Join` for you. Then you wouldn't have to import `filepath` all over the place.